docs/implement/user-attributes/README.md

```diff
@@ -72,6 +57,33 @@ of the more common ways is to declare them in a Spring XML configuration file wi
         </property>
     </bean>
 
+    <!-- CachingPersonAttributeDaoImpl Bean Definition -->
```

Comment:
I think that this would be better located in `docs/implement/user-attributes/jdbc.md`. Additionally, the `myunivCachingPersonDbJdbcAttributeSource` example bean should be brought over as well.